### ISSUE TITLE
Allman brace-style rule not working for switch-case blocks

### ISSUE THREAD

**Issue by enessoylu** [buzinas/tslint-eslint-rules#331]
tslint version: 5.10.0
typescript version: 2.8.1

tslint.json configuration related with the issue:
```js
 "brace-style": [
     true,
    "allman",
        {
            "allowSingleLine": true
        }
]
```

typescript code being linted:
```js
function test(str: string)
{
    switch(str) {
    case 'xyz':
        break;
    }
}
```

actual behavior.

Gives no error.

expected behavior.

Error on the starting of the switch-case block, saying that the opening curly brace should be on the next line.
